Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationOnboardingIcon.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ReferralsInvitationOnboardingIcon.swift
+//  project
+//
+//  Created by Sam Pettersson on 2019-06-03.
+//
+
+import Foundation
+import Flow
+import UIKit
+
+struct ReferralsInvitationOnboardingIcon {}
+
+extension ReferralsInvitationOnboardingIcon: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Disposable) {
+        let view = Icon(icon: Asset.clock, iconWidth: 16)
```

Comment:
Did the real-time clock become a mess? 🕐

Reply:
Haha yes, wasn't really worth spending time on 😅